docs/frontend/overview.rst

```diff
@@ -0,0 +1,172 @@
+Overview
+========
+
+The adhocracy 3 frontend is an opinionated web framework based on
+angular.js and written in TypeScript. Its main goal is to provide all
+the building blocks that developers need to implement participation
+processes of all kinds. The user interface should be consistent and
+recognizable while providing enough flexibility for a wide range of
+target audiences as well as branding.
+
+Angular
+-------
+
+`AngularJS <https://angularjs.org/>`__ is an open source JavaScript
+framework mainly developed by google.
+
+In contrast to traditional frameworks like jQuery, you do not directly
+interact with the DOM. Instead, you only interact with data structures
+in JavaScript. The DOM is *bound* to these data structures and updates
+automatically.
+
+Other notable features are *services* (singleton objects), *dependency
```

Comment:
I think the paragraph ist not helpful. For beginners it's too technical and for non-beginners it's too unclear. What do I know about angular services except that they are singleton? It does not convey much information.

I would suggest either make a small general description of the particularities of angular (directives allow to extend HTML for example) or link to a good intro.